### ISSUE TITLE
Improper headings throughout

### ISSUE THREAD

**Issue by MLCochrane** [MLCochrane/gosyp#134]
**Describe the bug**
Headings are inconsistent, missing when they should be present, or following improper order. The typography was never really brought out of the prototype phase making the page much more inaccessible than it should be.

**Expected behavior**
Headings should outline the page and follow correct levels.

**Additional context**
Along with correcting the current headings, additional headings should be added to outline the page and provide more context for the content. For example, a simple heading of "room details" above the sidebar content would go a long way for users whether or not they're using assistive tech.
